ansible/roles/products/common/docker-container/tasks/create-container/no-setup-no-terminator.yml

```diff
@@ -8,6 +8,16 @@
   set_fact:
     shared_volumes: "{{ common_shared_volumes + container_type_shared_volumes + custom_shared_volumes }}"
 
+- name: Find out if we should record all data from these hands
+  set_fact:
+    record_hand_data: "enabled"
+  when: config_branch in bag_all_data_from
+
+- name: Find out if we shouldn't record all data from these hands
+  set_fact:
+    record_hand_data: "disabled"
```

Comment:
this should be the default as variable and no need to set it as fact if not true